### PR TITLE
OP-681 & OP-766 summary table more exhaustive-remove type in summary filter by type

### PR DESCRIPTION
**Pull request by tsognong**

### CODE REVIEW

File: src/components/accessories/patientOPD/patientOPDTable/PatientOPDTable.tsx [hunk 1]
```diff
@@ -77,16 +77,16 @@ const PatientOPDTable: FunctionComponent<IOwnProps> = ({
...
           tableHeader={header}
           labelData={label}
           columnsOrder={order}
           rowsPerPage={5}
-          onDelete={onDelete}
+          onDelete={handleDelete ? onDelete : undefined}
           isCollapsabile={true}
-          onEdit={onEdit}
-          onView={onEView}
+          onEdit={handleEdit ? onEdit : undefined}
```
Comment (axelpezzo) [inline src/components/accessories/patientOPD/patientOPDTable/PatientOPDTable.tsx:89]: Is not enought don't pass the prop --and make it mandatory?

File: src/components/accessories/table/Table.tsx [hunk 2]
```diff
@@ -167,6 +168,7 @@ const Table: FunctionComponent<IProps> = ({
     <>
       <TableContainer component={Paper}>
         <MaterialComponent className="table" aria-label="simple table">
+          {title && <caption className="caption"> {title}</caption>}
```
Comment (axelpezzo) [inline src/components/accessories/table/Table.tsx:171]: Use an headline tag here (like h5 or h6)

**Comment by tsognong** [comment]
> @tsognong is this still in progress? Because I think that the work of the 766 is not completed yet, right?

Sorry @Alex, I didnt apply the remarks yet.

File: src/components/accessories/patientTherapy/patientTherapyTable/PatientTherapyTable.tsx [hunk 3]
```diff
@@ -113,16 +112,16 @@ const PatientTherapyTable: FunctionComponent<IOwnProps> = ({
...
               <Table
+                title={`${t("summary.therapy")} (${data.length})`}
                 rowData={formatDataToDisplay(data)}
                 compareRows={dateComparator}
                 tableHeader={header}
                 labelData={label}
                 columnsOrder={order}
                 rowsPerPage={5}
-                onDelete={onDelete}
+                onDelete={handleDelete ? onDelete : undefined}
```
Comment (alefalezza) [inline src/components/accessories/patientTherapy/patientTherapyTable/PatientTherapyTable.tsx:122]: Remove the conditional

File: src/components/accessories/patientTherapy/patientTherapyTable/PatientTherapyTable.tsx [hunk 4]
```diff
@@ -113,16 +112,16 @@ const PatientTherapyTable: FunctionComponent<IOwnProps> = ({
...
                 isCollapsabile={true}
-                onEdit={onEdit}
-                onView={onEView}
+                onEdit={handleEdit ? onEdit : undefined}
```
Comment (alefalezza) [inline src/components/accessories/patientTherapy/patientTherapyTable/PatientTherapyTable.tsx:124]: Remove conditional

File: src/components/accessories/patientTriage/patientTriageTable/PatientTriageTable.tsx [hunk 5]
```diff
@@ -93,13 +94,14 @@ const PatientTriageTable: FunctionComponent<IOwnProps> = ({
...
               <Table
+                title={`${t("summary.triage")} (${data.length})`}
                 rowData={formatDataToDisplay(data)}
                 compareRows={dateComparator}
                 tableHeader={header}
                 labelData={label}
                 columnsOrder={order}
                 rowsPerPage={5}
-                onDelete={onDelete}
+                onDelete={handleDelete ? onDelete : undefined}
```
Comment (alefalezza) [inline src/components/accessories/patientTriage/patientTriageTable/PatientTriageTable.tsx:104]: Remove conditional

**Pull request closed without merging**